srv/salt/ceph/remove/openattic/default.sls

```diff
@@ -9,8 +9,3 @@ remove openattic auth:
     - name: "ceph auth del client.openattic"
 
 {% endif %}
-
-fix salt job cache permissions:
-  cmd.run:
-  - name: "find /var/cache/salt/master/jobs -user root -exec chown {{ salt['deepsea.user']() }}:{{ salt['deepsea.group']() }} {} ';'"
-
```

Comment:
Why drop this?

Reply:
I'm fairly certain that salt2018 has fixed this issue internally. We can remove all other occurrences if that statement asserts true.

Reply:
Oh, cool. If that's fixed internally to salt I'll be very happy.